utensor_cgen/code_generator.py

```diff
@@ -41,15 +48,19 @@ def generate(self, src_fname):
 
     opFactory = OperatorFactory()
 
-    print("Parsing {}".format(self.pb_file))
-    ops_info, ops_torder, output_nodes = parse_pb(self.pb_file, self.output_nodes)
-    construct_order = Optimizer.optimize(ops_info, ops_torder, output_nodes, self.method)
+    print("Transforming graph: {}".format(self.pb_file))
+    new_pb_file = self._transform_graph(self.pb_file)
```

Comment:
So the cli now gets the ability to deal with dropout + quantation transform.

Reply:
Yes.
It will convert the pb file as following:
1. transform the original graph, like remove dropout layer or batch norm layer (future work)
2. quantize the transformed graph
3. convert quantized graph to cpp files